common/model/opening-hours.js

```diff
@@ -1,77 +1,103 @@
 // @flow
 function createPlace(data: Place) { return (data: Place); }
 
-type OpeningHoursDay = {|
-  dayOfWeek: string,
+type Days = 'Monday' | 'Tuesday' | 'Wednesday' | 'Thursday' | 'Friday' | 'Saturday' | 'Sunday';
+
+export type OpeningHoursDay = {
+  dayOfWeek: Days,
   closes: string,
   opens?: string,
   note?: string
-|};
+};
```

Comment:
Can we please use exact types?
https://flow.org/en/docs/types/objects/#toc-exact-object-types

Reply:
We can, think I did this because flow complained when I used intersection types with exact types

Reply:
Yea, I think you need to use exact for both  - I've done intersection typing (with a property you can enumerate on) here: https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/model/contributors.js

Reply:
👍